go/ct/spc/specification.go

```diff
@@ -2104,12 +2103,15 @@ func getRulesForCall(op OpCode, revision Revision, warm, zeroValue bool, opEffec
 
 func callEffect(s *st.State, addrAccessCost vm.Gas, op OpCode) {
 	isDelegateCall := op == DELEGATECALL
+	isCallCode := op == CALLCODE
 
 	gas := s.Stack.Pop()
 	target := s.Stack.Pop()
 	var value U256
-	if op == CALL {
+	if op == CALL || op == CALLCODE {
 		value = s.Stack.Pop()
+	} else if isDelegateCall {
+		value = s.Accounts.GetBalance(s.CallContext.AccountAddress)
```

Comment:
I think this is a left-over from an older version of the delegate call PR. Please re-sync and make sure that there are no merge issues.